src/components/mapgl/model/ValidGeomDrawPolygonMode.ts

```diff
@@ -13,7 +14,24 @@ ValidGeomDrawPolygonMode.fireInvalidGeom = function (feature) {
   });
 };
 
+ValidGeomDrawPolygonMode.fireOnClick = function () {
+  this.map.fire('draw.onClick', 'point drawn');
+};
+
+ValidGeomDrawPolygonMode.fireOnStop = function () {
+  this.map.fire('draw.onStop', 'draw end');
+};
+
+ValidGeomDrawPolygonMode.onTap = ValidGeomDrawPolygonMode.onClick = function(state, e) {
+  this.fireOnClick();
```

Comment:
`fireOnClick` is call if a click is done anywhere, should not it be called only if isn't a vertex ?

Reply:
fixed